advanced/rag_system.py

```
@@ -0,0 +1,295 @@
+                    'confidence': 0
+                })
+
+        print(f"\n✓ 완료!")
+        return {'categories': results}
+
+    def clear_database(self):
+        """Vector DB 초기화"""
+        try:
+            self.chroma_client.delete_collection(self.collection.name)
+            print(f"✓ 컬렉션 '{self.collection.name}' 삭제됨")
+        except:
```

Ruff (0.14.13)

Do not use bare `except`

(E722)